site/docs/components/dialog/examples.mdx

```diff
@@ -100,4 +100,11 @@ You can also render the `DialogCloseButton` as a child of the `Dialog` component
     - This should be used to force a user to action something within the dialog
     </LivePreview>
 
+      <LivePreview componentName="dialog" exampleName="DisableScrim" >
+    ### Disble Scrim
+
+    - Use the disableScrim prop to prevent the scrim from rendering
```

Comment:
nit, we should probably match all the list items to close with `.`

```suggestion
    - Use the disableScrim prop to prevent the scrim from rendering.
```